cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -108,6 +108,25 @@ spec:
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
         {{ end }}
+        {{ if .EnableVolumeReplicationSideCar }}
+        - name: volume-replication
+          image: {{ .VolumeReplicationImage }}
+          args :
+            - "--metrics-bind-address=0"
+            - "--leader-election-namespace={{ .Namespace }}"
+            - "--driver-name={{ .DriverNamePrefix }}rbd.csi.ceph.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--rpc-timeout=150s"
+            - "--health-probe-bind-address=:9998"
+            - "--leader-elect=false"
```

Comment:
this should be set to true and also add leader-election-namespace https://github.com/kube-storage/volume-replication-operator/blob/main/main.go#L68 which is pod namespace. check other sidecars for example